index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
Never mind I've been working to long clearly. I had left around the extra `exec` in there so it was actually trying to start nw 2x's I just didn't notice ::doh::